modules/secrets-store-azure.adoc

```diff
@@ -0,0 +1,278 @@
+  kind: ClusterRole
+  name: csi-secrets-store-provider-azure-cluster-role
+subjects:
+- kind: ServiceAccount
+  name: csi-secrets-store-provider-azure
+  namespace: openshift-cluster-csi-drivers
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  namespace: openshift-cluster-csi-drivers
+  name: csi-secrets-store-provider-azure
+  labels:
+    app: csi-secrets-store-provider-azure
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: csi-secrets-store-provider-azure
+  template:
+    metadata:
+      labels:
+        app: csi-secrets-store-provider-azure
+    spec:
+      serviceAccountName: csi-secrets-store-provider-azure
+      hostNetwork: true
+      containers:
+        - name: provider-azure-installer
+          image: mcr.microsoft.com/oss/azure/secrets-store/provider-azure:v1.4.1
+          imagePullPolicy: IfNotPresent
+          args:
+            - --endpoint=unix:///provider/azure.sock
+            - --construct-pem-chain=true
+            - --healthz-port=8989
+            - --healthz-path=/healthz
+            - --healthz-timeout=5s
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 8989
+            failureThreshold: 3
+            initialDelaySeconds: 5
+            timeoutSeconds: 10
+            periodSeconds: 30
+          resources:
+            requests:
+              cpu: 50m
+              memory: 100Mi
+            limits:
+              cpu: 50m
+              memory: 100Mi
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - mountPath: "/provider"
+              name: providervol
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: type
+                operator: NotIn
+                values:
+                - virtual-kubelet
+      volumes:
+        - name: providervol
+          hostPath:
+            path: "/var/run/secrets-store-csi-providers"
+      tolerations:
+      - operator: Exists
+      nodeSelector:
+        kubernetes.io/os: linux
+----
++
+[NOTE]
+====
+This configuration is modified from the configuration provided in the link:https://azure.github.io/secrets-store-csi-driver-provider-azure/docs/getting-started/installation/[Azure documentation] so that it works properly with {product-title}.
+====
++
+////
+TODO: Modified the above YAML with the following:
+
+- Added the ClusterRole and ClusterRoleBinding
+- Added namespace to the serviceaccount and daemonset
+- Changed securityContext to privileged on the container
+
+See if all of these are correct or not. Removed the "oc adm policy add-scc-to-use" step
+////
+
+.. Create the provider resources by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f azure-provider.yaml
+----
+
+. Create a secret provider class to define your secrets store provider:
```

Comment:
Hi @bergerhoffer, before creating secretproviderclass, we need to create azure keyvault. Did we mention the steps for creation of azure keyvault and setting parameters or i missed the doc?

Reply:
@ropatil010 So that is done in Azure key vault, correct?

We don't document any third-party items in our documentation, so they would have to rely on Azure's documentation on how to do that. 

In one of the prerequisites, I say "You have configured Azure Key Vault to store the required secrets." 

Is that sufficient to cover this, or is there any additional information we should mention in this prerequisite?

Reply:
Got it, thanks.!